src/internal/features/NeoRgbwwxFeature.h

```diff
@@ -0,0 +1,74 @@
+/*-------------------------------------------------------------------------
+NeoRgbwwxFeature provides feature classes to describe color order and
+color depth for NeoPixelBus template class
+
+Written by Michael C. Miller.
+
+I invest time and resources providing this open source code,
+please support me by dontating (see https://github.com/Makuna/NeoPixelBus)
+
+-------------------------------------------------------------------------
+This file is part of the Makuna/NeoPixelBus library.
+
+NeoPixelBus is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
+
+NeoPixelBus is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with NeoPixel.  If not, see
+<http://www.gnu.org/licenses/>.
+-------------------------------------------------------------------------*/
+#pragma once
+
+class NeoRgbwwxFeature : public Neo6Byte5xElementsNoSettings
+{
+public:
+    static void applyPixelColor(uint8_t* pPixels, uint16_t indexPixel, ColorObject color)
+    {
+        uint8_t* p = getPixelAddress(pPixels, indexPixel);
+
+        *p++ = color.R;
```

Comment:
of course, you mentioned the order is actually Bgrww so this order here will need adjusting.
Here is brain bender (but not critical), what if the cool verses warm were reversed?  RgbwwColor is device dependent, so there its abvious with CW and WW members.  Maybe features, the single letter abreviation should be w=warm or generic white, c = cool, n = nuetral?